piety-promoted-v3/modernized/03-chapter-03.adoc

```diff
@@ -338,9 +338,12 @@ He reached Swansea the 8th of the Second month, 1749, and died the 14th of the s
 aged about sixty-seven years, and a minister about fifty-three years;
 and as he lived so he died, a pattern of meekness and innocency.
 
-Benjamin`' Home`'s Last Legacy, or Serious Advice; recommended by him,
-a little before his decease, to the youth under the tuition of several schoolmasters.`'
+[.embedded-content-document.epistle]
```

Comment:
Not sure if this is truly an epistle?

Reply:
I'm not totally sure either, but I think you made the right call, let's leave it as an epistle.